packages/grid/src/HeaderCell.tsx

```diff
@@ -30,25 +38,77 @@ export function HeaderCell<T>(props: HeaderCellProps<T>) {
   const { ref, isFocusableContent, onFocus } =
     useFocusableContent<HTMLTableHeaderCellElement>();
 
+  const { onClickSortColumn, setSortByColumnId, sortOrder, sortByColumnId } =
+    useColumnSortContext();
+
+  const valueAlignRight = align === "right";
+
+  interface HeaderCellSortingIconProps {
+    justify: FlexContentAlignment;
+  }
+
+  const HeaderCellSortingIcon = ({ justify }: HeaderCellSortingIconProps) => {
+    const className = withBaseName("sortable");
+    const icon = (
+      <FlexLayout className={className} justify={justify} aria-hidden>
+        {sortOrder === "asc" && <ArrowUpIcon />}
+        {sortOrder === "desc" && <ArrowDownIcon />}
+      </FlexLayout>
+    );
+
+    return icon;
+  };
+
+  const ariaSort =
```

Comment:
This would be clearer as a map or a switch e.g.

```ts
const map = {
"asc": "ascending",
"desc": "descending"
}

map[sortOrder] || "none"
```

Reply:
updated to use a map